### DISCUSSION TITLE
where can  i download Chinese wubi language pack for 0.4.0 Beta alpha06(92)?

### DISCUSSION THREAD

**Discussion by han-fan** [florisboard/florisboard#2401] [Q&A]
thanks to you all for help.

i just want to download the following input method.
中文 (中国) [五笔98] / Chinese (China) [WUBI98]
中文 (中国) [五笔98-拼音混打] / Chinese (China) [WUBI98PINYIN]
中文 (中国) [五笔98-单字] / Chinese (China) [WUBI98SINGLE]
中文 (中国) [五笔-大字库] / Chinese (China) [WUBILARGE]

but i can not find the download link in this page:
https://github.com/florisboard/florisboard/blob/master/LANGUAGEPACKS-CHINESE.md
it seems that the link is missing from this page .

could anyone send me the languagepack or link?
thanks!
i installed the lasted 0.4.0. beta

**Comment by moonbeamcelery** [comment]
Apologies. Please use this link as a temporary measure.
https://drive.proton.me/urls/CRWFK2GCAR#zLjVhpYR3fo8
